tests/e2e/e2e_exec_test.go

```
@@ -0,0 +1,956 @@
+	stdOut := []byte{}
+	scanner := bufio.NewScanner(&outBuf)
+	for scanner.Scan() {
+		stdOut = scanner.Bytes()
+		var out map[string]interface{}
+		err = json.Unmarshal(stdOut, &out)
+		s.Require().NoError(err)
+		if err != nil {
+			return nil, fmt.Errorf("hermes relayer command returned failed with error: %s", err)
+		}
+		// errors are caught by observing the logs level in the stderr output
+		if lvl := out["level"]; lvl != nil && strings.ToLower(lvl.(string)) == "error" {
```

golangci-lint (1.62.2)

type assertion must be checked

(forcetypeassert)